go/libkb/naclwrap.go

```diff
@@ -793,6 +793,25 @@ func DeriveSymmetricKey(inKey NaclSecretBoxKey, reason EncryptionReason) (NaclSe
 	return outKey, nil
 }
 
+func DeriveNonReversed(inKey [32]byte, reason DeriveReason) (outKey [32]byte, err error) {
```

Comment:
This is poorly named. This and `DeriveSymmetricKey` do basically the same job but in different ways.